docs/src/systems/fluid.md

```diff
@@ -1,10 +1,138 @@
+This is achieved by incorporating forces that replicate the viscous stress term found in the Navier–Stokes equations.
+As a result, the method is particularly effective for simulating low-speed, incompressible, or weakly compressible flows,
+where it is crucial to capture the actual behavior of the fluid.
+
+### Model comparison
+
+#### ArtificialViscosityMonaghan
+
+ArtificialViscosityMonaghan by Monaghan ([Monaghan1992](@cite), [Monaghan1989](@cite))
+is designed primarily for compressible,
+high-speed flows where shock capturing is critical.
+In its implementation, the method includes a dissipation term
+that increases when particles approach each other.
+This increase in dissipation is triggered by the relative motion between particles:
+as particles come closer and compress the local flow,
+the artificial viscosity term becomes stronger to damp out rapid changes
+and prevent unphysical clustering.
+This ensures that while the simulation remains stable in challenging
+flow regimes with large density or pressure variations,
+the physical behavior is not overly altered.
+
+##### Mathematical Formulation
+
+The artificial viscosity between two particles ``a`` and ``b`` is given by:
+
+```math
+\Pi_{ab} =
+\begin{cases}
+    -\frac{\alpha c \mu_{ab} + \beta \mu_{ab}^2}{\bar{\rho}_{ab}} & \text{if } v_{ab} \cdot r_{ab} < 0, \\
+    0 & \text{otherwise}
+\end{cases}
+```
+
+where:
+
+- **``\alpha`` and ``\beta``** are viscosity parameters,
+- **``c``** is the local speed of sound,
+- **``\bar{\rho}_{ab}``** is the arithmetic mean of the densities of particles ``a`` and ``b``.
+
+The term ``\mu_{ab}`` is defined as:
+
+```math
+\mu_{ab} = \frac{h \, v_{ab} \cdot r_{ab}}{\Vert r_{ab} \Vert^2 + \epsilon h^2},
+```
+
+with:
+
+- **``h``** being the smoothing length,
+- **``\epsilon``** a small parameter to prevent singularities,
+- **``r_{ab} = r_a - r_b``** representing the displacement vector between particles,
+- **``v_{ab} = v_a - v_b``** representing the relative velocity between particles.
+
+##### Resolution Dependency and Effective Viscosity
+
+To ensure that the simulation maintains a consistent Reynolds number when the resolution changes, the parameter ``\alpha`` must be adjusted accordingly. Monaghan (2005) introduced an effective physical kinematic viscosity ``\nu`` defined as:
+
+```math
+\nu = \frac{\alpha h c}{2d + 4},
+```
+
+where **``d``** is the number of spatial dimensions. This relation allows the calibration of ``\alpha`` to achieve the desired viscous behavior as the resolution or simulation conditions vary.
+
+#### ViscosityMorris
+
+ViscosityMorris is ideal for moderate to low Mach number flows where accurately modeling physical viscous behavior is essential. Developed by [Morris (1997)](@cite Morris1997) and later applied by [Fourtakas (2019)](@cite Fourtakas2019), this method directly simulates the viscous stresses found in fluids rather than relying on artificial viscosity.
```

Comment:
```suggestion
`ViscosityMorris` is ideal for moderate to low Mach number flows where accurately modeling physical viscous behavior is essential. Developed by [Morris (1997)](@cite Morris1997) and later applied by [Fourtakas (2019)](@cite Fourtakas2019), this method directly simulates the viscous stresses found in fluids rather than relying on artificial viscosity.
```